src/ape_plugins/utils.py

```diff
@@ -22,7 +22,7 @@
     }
 
 else:
-    logger.warning("$GITHUB_ACCESS_TOKEN not set, skipping 2nd class plugins")
+    logger.warning("$GITHUB_ACCESS_TOKEN not set, unable to list plugins")
```

Comment:
```suggestion
    logger.warning("$GITHUB_ACCESS_TOKEN not set, unable to list all plugins")
```

Reply:
What about `unable to list trusted plugsin` ?

Reply:
sure

Reply:
I only say that because this gets displayed without the `-a`, it is a little weird with `all`.

`ape plugins list`

Output:
`GITHUB_ACCESS_TOKEN not set, unable to list all plugins`